conanfile.py

```diff
@@ -7,13 +7,15 @@
 class MongoCDriverConan(ConanFile):
     name = "mongo-c-driver"
     version = "1.9.4"
-    url = "http://github.com/bincrafters/conan-mongo-c-driver"
-    description = "A high-performance MongoDB driver for C "
-    license = "https://github.com/mongodb/mongo-c-driver/blob/{0}/COPYING".format(version)
+    url = "https://github.com/mongodb/mongo-c-driver"
+    description = "A high-performance MongoDB driver for C"
+    topics = ["conan", "libmongoc", "mongodb"]
+    author = "Bincrafters <bincrafters@gmail.com>"
+    license = "Apache-2.0"
     settings =  "os", "compiler", "arch", "build_type"
     options = {"shared": [True, False]}
-    default_options = "shared=False"
-    requires = 'zlib/[~=1.2]@conan/stable'
+    default_options = {"shared": False}
+    requires = 'zlib/[>= 1.2.11]@conan/stable'
```

Comment:
```suggestion
    requires = 'zlib/1.2.11@conan/stable'
```

Please don't use version ranges (Conan team recommendation, Bincrafters convention)

Reply:
Forgive me if I'm being a bit dense, but why don't we want ranges here? Doesn't specifying a particular version open the door to dependency version conflicts?

Reply:
Version ranges are slow to resolved and shouldn't be used for performance reason and also, maybe even more important, for reproducible builds you need absolute versions.

If you use in the parent project e.g. zlib/1.2.11 and a dependencies of ours it using by itself zlib/1.2.9, then Conan will override it to use zlib/1.2.11, which means that you end up in the same situation as you would use a version range in the first place. Including possible incompatibilities as it can happen with version ranges or no conflict at all. However, better performance and reproducible.